install.sh

```diff
@@ -382,6 +396,7 @@ Options:
 --python, -p                python version (only python3 versions are supported)
 --with-qt                   build the Qt GUI
 --without-qt                don't build the Qt GUI
+--docker-install system wide install as root for minimal Docker installs
```

Comment:
Alphabetical ordering and spacing.
```suggestion
--docker-install            system wide install as root for minimal Docker installs
--python, -p                python version (only python3 versions are supported)
--with-qt                   build the Qt GUI
--without-qt                don't build the Qt GUI
```